tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -134,3 +128,220 @@ def test_positive_deploy_configure_by_script(session, form_data):
         assert session.contenthost.search(guest_name)[0]['Subscription Status'] == 'green'
         session.virtwho_configure.delete(name)
         assert not session.virtwho_configure.search(name)
+
+
+@tier2
+def test_positive_debug_option(session, form_data):
+    """ Verify debug checkbox and the value changes of VIRTWHO_DEBUG
+
+    :id: d6f9940f-c003-4099-97bf-4d8820ab710e
+
+    :expectedresults:
+        1. if debug is checked, VIRTWHO_DEBUG=1 in /etc/sysconfig/virt-who
+        2. if debug is unchecked, VIRTWHO_DEBUG=0 in /etc/sysconfig/virt-who
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        deploy_configure_by_command(config_command)
+        assert get_configure_option('VIRTWHO_DEBUG', VIRTWHO_SYSCONFIG) == '1'
+        session.virtwho_configure.edit(name, {'debug': False})
+        deploy_configure_by_command(config_command)
+        assert get_configure_option('VIRTWHO_DEBUG', VIRTWHO_SYSCONFIG) == '0'
+        session.virtwho_configure.delete(name)
+        assert not session.virtwho_configure.search(name)
+
+
+@tier2
+def test_positive_interval_option(session, form_data):
+    """ Verify interval dropdown options and the value changes of VIRTWHO_INTERVAL.
+
+    :id: 412a01eb-ae9c-4b23-aeb5-2595873b27e2
+
+    :expectedresults:
+        VIRTWHO_INTERVAL can be changed in /etc/sysconfig/virt-who if the
+        dropdown option is selected to Every 2/4/8/12/24 hours, Every 2/3 days.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        intervals = {
+            'Every hour': '3600',
+            'Every 2 hours': '7200',
+            'Every 4 hours': '14400',
+            'Every 8 hours': '28800',
+            'Every 12 hours': '43200',
+            'Every 24 hours': '86400',
+            'Every 2 days':  '172800',
+            'Every 3 days':  '259200',
+        }
+        for option, value in sorted(intervals.items(), key=lambda item: int(item[1])):
+            session.virtwho_configure.edit(name, {'interval': option})
+            deploy_configure_by_command(config_command)
```

Comment:
same as above .. add assertion for UI end too.

Reply:
as above

Reply:
same as above ..